src/foam/u2/view/MultiSelectView.js

```diff
@@ -0,0 +1,202 @@
+      border: 1px solid $grey400;
+      box-shadow: none !important;
+      border-radius: 4px;
+      margin-top: 5px;
+  }
+  ^choices-holder {
+      display: flex;
+      flex-direction: column;
+      gap: 2rem;
+      line-height: 1.2;
+      padding: 5px;
+  }
+  ^input-holder {
+      display: flex;
+      align-items: center;
+      gap: 1rem;
+      color: $black;
+      width: 100%;
+  }
+  ^checkbox-input {
+      -webkit-appearance: none;
+      appearance: none;
+      border-radius: 2px;
+      border: solid 1px $grey400;
+      height: 1.275em;
+      margin: 7px 0;
+      padding: 0px;
+      transition: background-color 140ms, border-color 140ms;
+      width: 1.275em;
+      flex-shrink: 0;
+      cursor: pointer;
+  }
+  ^checkbox-input:checked {
+      background-color: $primary500;
+      border-color: $primary500;
+  }
+  ^checkbox-input:checked:disabled {
+      border-color: $grey600;
+      background-color: $grey600;
+      fill: white;
+  }
+  ^checkbox-input:checked:after {
+      position:relative;
+      top: 1px;
+      content: url("/images/checkmark-white.svg");
+  }
+  `,
+
+  requires: [
+      'foam.u2.PopupView'
+  ],
+
+  messages: [
+      { name: 'OPTIONS_MSG', message: 'options' },
+      { name: 'CHOOSE_1_OF_FOLLOWING_OPTIONS', message: 'Choose one of the following options' },
+      { name: 'CHOOSE_AT_LEAST_1_OPTION', message: 'Choose at least one option' },
+      { name: 'CHOOSE_AT_LEAST', message: 'Choose at least' },
+      { name: 'CHOOSE_EXACTLY', message: 'Choose exactly' },
+      { name: 'CHOOSE', message: 'Choose' }
+  ],
+
+  properties: [
+      {
+          name: 'placeholder'
+      },
+      {
+          class: 'Function',
+          name: 'onSelect'
+      },
+      {
+          name: 'choices',
+          documentation: `
+              An array of choices which are single choice is denoted as [value, label, isFinal]
+          `,
+          factory: function() {
+              return [];
+          },
+          adapt: function(old, nu) {
+              if ( foam.Object.isInstance(nu) ) {
+                  var out = [];
+                  for ( var key in nu ) {
+                    if ( nu.hasOwnProperty(key) ) out.push([ key, nu[key] ]);
+                  }
+                  return out;
+              }
+      
+              nu = foam.Array.shallowClone(nu);
+      
+              // Upgrade single values to [value, value].
+              for ( var i = 0; i < nu.length; i++ ) {
+                  if ( ! Array.isArray(nu[i]) ) {
+                    nu[i] = [ nu[i], nu[i] ];
+                  }
+              }
+              return nu;
+          }
+      },
+      {
+          name: 'data',
+          factory: function() {
+            return [];
+          },
+      },
+  ],
+
+  actions: [
+    {
+      name: 'options',
+      icon: '/images/chevron-down.svg',
+      code: async function() {
+        self = this;
+        let selectCreateModal = this.PopupView.create({
```

Comment:
PopupView is now deprecated and doesnt work responsively mobile screens. You can use foam.u2.md.OverlayDropdown instead